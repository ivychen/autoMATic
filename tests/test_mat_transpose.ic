bool doubleTrans(matrix M) {
	matrix N;
	N = M'';

	if (N == M) {
		return true;
	}

	return false;
}
