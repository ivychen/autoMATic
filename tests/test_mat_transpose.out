
bool doubleTrans(M)
{
matrix N;
N = ''M;
if (N == M)
{
return true;
}
return false;
}
